src/main/riddl/Common/Contact.riddl

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) 2022 Improving Inc.
 
 // Types for Contact Information
-type MobileNumber is Pattern("\\(([0-9]{3})\\)([0-9]{3})-([0-9]{4})")
+type MobileNumber is Pattern("(([0-9]{3}))([0-9]{3})-([0-9]{4})")
```

Comment:
It must have been something wrong with version 0.22 because that was the only way I could get it to compile. Now it seems to be working the proper way. Will push the fix